src/reactjs/components/DeviceGrid/DeviceGrid.tsx

```diff
@@ -76,6 +79,7 @@ export class DeviceGrid extends React.Component<gridProps, gridStates>{
     connectionStatusControl = () => {
         let protocol = window.location.protocol === "https:" ? "https" : "http";
         const updateConnectionSocket = new WebSocket(window.location.protocol.replace(protocol, "wss") + "//" + this.props.mpsServer + "/notifications/control.ashx")
```

Comment:
why does it matter what protocol the site is served on? Especially if its going to be replaced with "wss". Update this to be: 
```
const updateConnectionSocket = new WebSocket(`wss://${this.props.mpsServer}/notifications/control.ashx`)
```

Reply:
done